src/Compilers/CSharp/Test/Emit/Emit/CovariantReturnTests.cs

```diff
@@ -235,10 +243,11 @@ public sealed class PreserveBaseOverridesAttribute : Attribute { }
                 source,
                 options: options,
                 parseOptions: TestOptions.WithCovariantReturns,
-                references: references);
+                references: references,
+                assemblyName: assemblyName);
         }
 
-        [ConditionalFact(typeof(CovarantReturnRuntimeOnly))]
+        [ConditionalFact(typeof(CovariantReturnRuntimeOnly))]
```

Comment:
>[ConditionalFact(typeof(CovariantReturnRuntimeOnly))] [](start = 8, length = 53)

It is quite possible I am missing the motivation for conditionally disabling the tests. If it is just to avoid the failure because we are unable to execute the code in proc on some platforms, Default Interface Implementation tests solve this problem by conditionally passing null as ```expectedOutput``` to ```CompileAndVerify```. This way we get as much verification as possible on each platform.

Reply:
Doing so here would simply duplicate tests already in the symbol tests.

---
In reply to: [461782992](https://github.com/dotnet/roslyn/pull/46367#discussion_r461782992) [](ancestors = 461782992)